testing/variable.go

```diff
@@ -359,7 +359,7 @@ func CreateVariable(init func(VariableFields, *testing.T) (platform.VariableServ
 					Name:           "existing-variable",
 					Selected:       []string{"a"},
 					Arguments: &platform.VariableArguments{
-						Type:   "query",
+						Type:   "constant",
```

Comment:
this makes the marshaling invalid, the types are a mismatch, this fixes it to represent the correct variable. Weirdly no other service implementation noticed this 😳